package.json

```diff
@@ -239,12 +239,12 @@
     "stylelint-webpack-plugin": "^3.0.1",
     "svg-url-loader": "^7.1.1",
     "table": "^6.7.1",
-    "terser-webpack-plugin": "^5.2.5",
+    "terser-webpack-plugin": "^5.3.1",
     "url-loader": "^4.1.1",
-    "webpack": "^5.64.4",
+    "webpack": "^5.70.0",
```

Comment:
We have a lot of options and customizations with our webpack setup, I'm not super comfortable upgrading a critical dependency like this without testing the options and flags. I think it's OK with bug fixes (0.0.x), but minors (0.x.0) makes me a little uneasy.

Ideally, we need some automated way to test our webpack setup before upgrades (part of an overarching dependency management strategy), but at a minimum we should get a test plan for all the options: https://github.com/department-of-veterans-affairs/vets-website/blob/master/script/build-help.js so we aren't just relying on local testing and the e2e/unit tests teams write

Reply:
I did tested a few of those options. However, I went ahead and added all the test cases used in the Webpack 5 upgrade in the description above and make sure to test them all.